components/meter/stories/meter.test.js

```diff
@@ -28,6 +24,10 @@ export const MeterGroup = Variants({
 			testHeading: "Without value label",
 			showValueLabel: false,
 		},
+		{
+			testHeading: "Help text",
+			helpText: "Help text message to add more context",
+		},
```

Comment:
Do you think it's worth it to add the help text to the sizing test template, sort of like Rise did with tag group? The reason I'm asking is because the help text changes its t-shirt size when the meter does, so do we want to have a VRT baseline for that behavior?

<img width="974" alt="Screenshot 2025-06-25 at 4 58 48 PM" src="https://github.com/user-attachments/assets/ffd567b6-538b-49a3-8ed8-ad31e9d2e513" />

I know you reviewed tag group, but just in case: https://github.com/adobe/spectrum-css/pull/3966/files#diff-e4784e05af5c8f7ab43b11c1d8211a05747916857afebf6f74225141329178a6